packages/block-editor/src/store/reducer.js

```diff
@@ -67,8 +67,8 @@ function mapBlockOrder( blocks, rootClientId = '' ) {
  * @return {Object} Block order map object.
  */
 function mapBlockParents( blocks, rootClientId = '' ) {
-	return blocks.reduce( ( result, block ) => Object.assign(
-		result,
+	return blocks.reduce( ( resultAccumulator, block ) => Object.assign(
```

Comment:
It seems like `resultAccumulator` could be simplified to just `accumulator`. They `result` prefix doesn't sound like it brings additional context here.